### ISSUE TITLE
Reduce number of automatic checks with travis

### ISSUE THREAD

**Issue by gplssm** [openego/eGon-data#41]
Right now, in a PR, 4 checks with Travis are triggered.

![grafik](https://user-images.githubusercontent.com/8255114/98543567-fa9b8d80-2292-11eb-98b2-4feb06c6bc3c.png)

I'd suggest to keep only one. This one should be "Travis CI - Pull Request".

**Comment by gnn** [comment]
Just one thing: the PR and the Branch checks are actually different. The latter runs the build with the current branch checked out, i.e. it tests the current branch. The PR check tests the state that the repository would be in after merging the current branch into the target branch. Just wanted to point that out. If you still want to only keep the PR check, that's fine though.
Regardless, the `continuous-integration` checks are probably just duplicates of the other two, so I'm going to remove those.

**Comment by gplssm** [comment]
> Just one thing: the PR and the Branch checks are actually different. The latter runs the build with the current branch checked out, i.e. it tests the current branch. The PR check tests the state that the repository would be in after merging the current branch into the target branch. Just wanted to point that out. If you still want to only keep the PR check, that's fine though.

Thanks for explaining the difference, even though I do not understand it to the very end.
Let's keep both, PR and Branch Travis checks.

> Regardless, the `continuous-integration` checks are probably just duplicates of the other two, so I'm going to remove those.

:+1:

**Comment by gnn** [comment]
Removed the `continuous-integration` checks before you even answered but had to wait to actually test it. Looking into PR #31, I see that my fix worked, we now only have two Travis checks remaining. Also, the PR is a nice example of both checks giving different answers. For the sake of completeness: the reason the duplicate checks was [Travis' move to GitHub Apps][0]. Since Travis was configured via the repository's `Settings -> Webhooks` but also via `Settings -> Integrations`, Travis was notified of repository changes twice. Removing the, basically deprecated, `Webhooks` entry fixed things.

[0]: https://blog.travis-ci.com/2018-05-07-announcing-support-for-github-checks-api-on-travis-ci-com

**Comment by gplssm** [comment]
I created a new draft PR https://github.com/openego/eGon-data/pull/61
There, Travis checks are missing now entirely. Did you remove all of them?

**Comment by gnn** [comment]
Nope. Didn't change a thing.
But apparently Travis changed their [pricing][0] [model][1]. Checking out the [project's page][2], I see a notification on the top stating that:

> Builds have been temporarily disabled for public repositories due to a negative credit balance. Please go to the [Plan page][3] to replenish your credit balance or alter your [Consume paid credits for OSS setting][4].

So now it seems that builds are metered via credits. We exceeded the credits initially given on the first of November and now we have to apply for new, maybe renewable, OSS credits. Or buy paid ones...
I suggest disabling macOS builds because they are the most expensive in terms of credits, and applying for renewable OSS credits once this is done.

[0]: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
[1]: https://blog.travis-ci.com/oss-announcement
[2]: https://www.travis-ci.com/github/openego/eGon-data
[3]: https://www.travis-ci.com/organizations/openego/plan
[4]: https://www.travis-ci.com/organizations/openego/plan?anchor=oss-consumption

**Comment by gnn** [comment]
Closing in favour of #63.
